src/atoms/Text/Text.stories.js

```diff
@@ -19,4 +19,22 @@ storiesOf('Text', module)
     <Text size={select('Size', sizeOptions, 'large')}>
       {text('Display Text', 'The quick brown fox jumps over the lazy dog')}
     </Text>
+  ))
+  .add('as span', () => (
+    <Text size={select('Size', sizeOptions, 'large')} as="span">
+      {text('Display Text', 'The quick brown fox jumps over the lazy dog')}
+    </Text>
+  ))
+  .add('show content in only 3 line and hide the rest ', () => (
```

Comment:
Add maxLines as a knob

Reply:
i have done that  in the separate story. above story shows  how to use 'as' prop

Reply:
"show content in only 3 line and hide the rest " looks odd on mobile :(. The text is too long. We can skip this story and let the user play with the knobs if they need to.

Reply:
@susheelachoudry I think you missed this one

Reply:
my bad,  done